### PR TITLE
cephfs admin: Fix CI issues due to recent changes in ceph nautilus

### PR DESCRIPTION
**Pull request by phlogistonjohn**
Apparently, the subvolumegroup snapshot feature is being removed from all active ceph versions
and will not be present in future versions. Since we won't be able to
test it any more, and the api is provisional still, we just remove it
entirely.

Fixes: #390 

Additionally, this ceph version brings in a change to the location of the snapshot directory in the file system. The existing code tried to anticipate this coming behavior chage but was not correct and missed a "uuid directory" in the path to the snapshot.

Fixes: #387 

## Checklist
- [x] Added tests for features and functional changes
- [x] Public functions and types are documented
- [x] Standard formatting is applied to Go code

### CODE REVIEW

**Comment by phlogistonjohn** [comment]
@ShyamsundarR  - FYI

**Review by nixpanic — APPROVED** [review]

File: cephfs/admin/workflow_test.go [hunk 1]
```diff
@@ -158,6 +158,8 @@ func TestWorkflow(t *testing.T) {
 	require.NoError(t, err)
 	require.NotNil(t, sinfo)
 
+	time.Sleep(500 * time.Millisecond) // is there a race?
```
Comment (agarwal-mudit) [inline cephfs/admin/workflow_test.go:161]: AFAIK, CephFS snap creation is not completely synchronous. So this can be expected.

**Review by agarwal-mudit — APPROVED** [review]
LGTM

File: cephfs/admin/workflow_test.go (hunk 1 above)
Comment (phlogistonjohn) [reply cephfs/admin/workflow_test.go:161]: Thanks! It's been pretty fast, but this will hopefully prevent future flakiness.

**Pull request merged**